ecs_deploy/ecs.py

```diff
@@ -360,9 +377,12 @@ def set_commands(self, **commands):
                 self._diff.append(diff)
                 container[u'command'] = self.parse_command(new_command)
 
-    def set_environment(self, environment_list, exclusive=False):
+    def set_environment(self, environment_list, env_file=((None, None),), exclusive=False):
```

Comment:
As this is optional, please add the new argument at the end to not unnecessarily change the signature of the function. Could help keeping it backwards compatible in one of the forks

Reply:
Ah, good point. I have changed it in the latest commit. I had to set env_file = explicitly in the tests now for some reason, otherwise the test functions thought the explicit boolean was the env_file touple. This doesn't really make sense to me, but the code is passing the tests now at least.

Reply:
Thanks for the change!

The reason is, that the function now expects the second argument to be the boolean for `exclusive_env`. You either have to provide all arguments to the function (without argument names) or to provide the argument name, if you don't provide all arguments which are defined in the function's signature.

Reply:
Ah, I see. Thanks!

I added the requested features in the latest commit.